zebra-chain/src/primitives/zcash_primitives.rs

```diff
@@ -22,7 +28,10 @@ impl TryFrom<&Transaction> for zcash_primitives::transaction::Transaction {
             Transaction::V5 {
                 network_upgrade, ..
             } => network_upgrade,
-            Transaction::V1 { .. } | Transaction::V2 { .. } |  Transaction::V3 { .. } |  Transaction::V4 { .. } => panic!("Zebra only uses librustzcash for V5 transactions"),
+            Transaction::V1 { .. }
+            | Transaction::V2 { .. }
+            | Transaction::V3 { .. }
+            | Transaction::V4 { .. } => panic!("Zebra only uses librustzcash for V5 transactions"),
```

Comment:
If you want to do all the sighashes through librustzcash, that would also be ok.

Let's open a follow-up ticket for that?

Reply:
Opened in #2183